drivers/gles3/shaders/stdlib_inc.glsl

```diff
@@ -39,6 +39,12 @@ vec2 unpackSnorm2x16(uint p) {
 	return clamp((v - 32767.0) * vec2(0.00003051851), vec2(-1.0), vec2(1.0));
 }
 
+#endif
+
+// These functions are not available in GLSL 330 or GLSL ES 300 so we need to
+// emulate them on all platforms. But for some reason they are exposed on
+// Android when using GLSL ES 300, so don't include them there.
+#ifndef ANDROID_ENABLED
```

Comment:
I think a safer bet here would be to expose these with a unique name, so they don't conflict with GLSL ES 310 or drivers which for whatever obscure reason implemented these in GLSL ES 300.

Reply:
That would just push the problem up the pipe as we would still need some logic to decide whether to use our non-conflicting name version, or the built-in version. I'll take a deeper look and see if we are accidentally enabling an extension or somehow otherwise exposing GLSL ES 310 functionality. I already verified that we are properly specifying version ES 300

Reply:
No this wouldn't be a problem here, as the built-in version should *never* be defined. We're limited by WebGL 2.0 to GLSL ES 300, so we'll never have access to the `4x8` variants. So you can just give them other names and they won't conflict with the ones weirdly present on Adreno.